vowpalwabbit/core/include/vw/core/cb.h

```diff
@@ -77,6 +77,21 @@ class cb_eval_label
 extern VW::label_parser cb_eval_label_parser_global;  // for evaluation of an arbitrary policy.
 }  // namespace VW
 
+namespace VW
+{
+class cb_with_observations_label

```

Comment:
May want to consider putting this label in its own file like ccb, slates, etc to not overcrowd cb.h